torch/onnx/_internal/exporter/_fx_passes.py

```diff
@@ -15,13 +15,20 @@ def decompose_with_registry(
 
     This function is needed so it shows clearly on the profiler results.
     """
+    from torch._decomp import decomp_table_to_post_autograd_aten
     onnx_registered_ops = set(_decomp.get_onnx_implemented_overloads(registry))
     decomp_table = _decomp.create_onnx_friendly_decomposition_table(onnx_registered_ops)
+    decomp_table.update(decomp_table_to_post_autograd_aten())
```

Comment:
Could you explain this line? Thanks

Reply:
Can it be part of create_onnx_friendly_decomposition_table ?

Reply:
Looks like everything will be decomped if it’s in the table now. Is that correct? If so feel free to maintain this code for now.